backend/mlarchive/archive/views.py

```diff
@@ -30,7 +30,7 @@
 from elasticsearch.exceptions import RequestError
 
 from mlarchive.utils.decorators import (check_access, superuser_only, pad_id, 
-    check_list_access)
+    check_list_access, staff_only)
```

Comment:
Why?
(I'm not going to block on this, but am curious why we think this needs controlled access?)